README.md

```diff
@@ -67,24 +67,12 @@ flowchart LR
 
 ## Getting Started
 
-The [Getting started with KinD][tut-kind] tutorial explains how to get started
+The [Getting Started](docs/getting-started.md) tutorial explains how to get started
```

Comment:
Simplified as the link doesn't need to be re-used. This was previously resulting in a 404 on GitHub.